content/docs/command-reference/exp/show.md

```diff
@@ -74,7 +74,8 @@ will be generated using the same data from the table.
 - `-A`, `--all-commits` - include experiments derived from all Git commits, as
   well as from the last one. This prints all experiments in the project.
 
-- `-n <num>`, `--num <num>` - show the last `num` commits from HEAD.
+- `-n <num>`, `--num <num>` - show experiments from the `<rev>` commit and `num`
+  commits before it. (`-1` for the whole history of the `<rev>`)
```

Comment:
Out of scope? This is being done in https://github.com/iterative/dvc.org/pull/3187 (please revert).